src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.getElementsByClassName('wall')[0].addEventListener('click', e => {
+  const spider = document.getElementsByClassName('spider')[0];
+  const wall = document.getElementsByClassName('wall')[0];
```

Comment:
It is better to take it outside the event listener, and call it from this variable.

Reply:
ok, thanks